azavea/raster-vision
# Add validate methods for config builders

## Overview

This PR ensures that proper error messages are thrown when calling config builders. These errors direct the user to the corresponding with_method, as opposed to an unclear 'missing positional argument' error.

### Checklist


## Testing Instructions

* Unit tests and integration tests will cover this PR. For each validate method, there are tests which ensure that a config error is thrown when missing required configs as well as tests which ensure that no config errors are thrown when required configs are given.

Closes #395 

Depends on #413
